src/VSIX/VSIX.csproj

```diff
@@ -136,15 +136,15 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Data.Framework, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.VisualStudio.Data.Framework.14.3.25407\lib\net45\Microsoft.VisualStudio.Data.Framework.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.VisualStudio.Data.Framework.14.3.25407\lib\net452\Microsoft.VisualStudio.Data.Framework.dll</HintPath>
```

Comment:
That's odd, this path doesn't actually exist - but Appveyor still produces a build. I'm guessing that this is only the hint path, and that a version of this assembly is found in the GAC instead - but I'm not sure that's a good thing. To be on the safe side it may be better to undo these changes.

We can also look into updating these VS assemblies, but it's best to tread very carefully around the VSIX plumbing.

Reply:
I may have been unclear, rather than removing the hint paths altogether, I'd rather we left them the way they were, referencing the `net45` versions.

Reply:
Just checking if they are needed at all (they are). Reverted both the removal and the changes now.